### PR TITLE
Yeah finally Sub2W2C3C2

### PR DESCRIPTION
**Pull request by bengrif**
Carefully created this one.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/manipulator/ManipulatorSubsystem.java [hunk 1]
```diff
@@ -213,6 +213,14 @@ public Command getPodiumShootCommand() {
     return getShootCommand(ArmPosition.PODIUM_SHOOT, ShooterVelocity.PODIUM_SHOOT);
   }
 
+  public Command getFarPodiumShootCommand() {
```
Comment (Kellenn) [inline src/main/java/frc/robot/subsystems/manipulator/ManipulatorSubsystem.java:216]: What is the difference between a far and non-far shot?

**Review by Kellenn — CHANGES_REQUESTED** [review]
Yay! 🎉🎉

**Comment by Kellenn** [comment]
I can commit my requested changes tomorrow.

**Comment by bengrif** [comment]
The auto still had a few misses in the last tests, I plan on testing it again tomorrow so I'll implement your suggestions.  Haven't decided if it's going to do a predetermined podium shot or autoshoot.

File: src/main/java/frc/robot/subsystems/manipulator/ManipulatorSubsystem.java (hunk 1 above)
Comment (bengrif) [reply src/main/java/frc/robot/subsystems/manipulator/ManipulatorSubsystem.java:216]: well the far shot is farther :)   It's like the podium shot but a little bit farther away.    But then when I went to tune, it was very close to podium anyway.

**Review by Kellenn — APPROVED** [review]

**Pull request merged**